src/migrations/index.js

```diff
@@ -0,0 +1,28 @@
+import {
+  getLastPackageVersion,
+  getPackageVersion,
+  setPackageVersion,
+} from '../local-settings'
+import { clearActivities } from './migrations'
+import { isNewVersion, satisifesRange } from './helper'
+
+const MIGRATIONS = {
+  '1.0.1': () => clearActivities(),
+}
```

Comment:
Not opposed to this format, but maybe an array of objects is a better fit?

Reply:
What's the clear advantage you see with the suggested format? 

Won't there be only one key value pair for each object anyways?

Reply:
> Won't there be only one key value pair for each object anyways?

Yep. I'm just always a bit hesitant to use something like `1.0.1` as a key—it's not wrong, but just feels a little odd.

It feels a bit more natural to do use one of these two:

```js
const MIGRATIONS = [
  ['1.0.1', () => {}],
]

const MIGRATIONS = [
  {
    version: '1.0.1',
    migration: () => {}
  },
]
```

The first is more similar to what we currently have, and reduces the `Object.keys().forEach()` into a `for of` loop, and the latter is more verbose and probably what would be used in a JSON configuration file.

Reply:
Thanks for the explanation!